parts/kubernetesmastercustomscript.sh

```diff
@@ -103,32 +109,75 @@ function setDockerOpts () {
     sed -i "s#^DOCKER_OPTS=.*#DOCKER_OPTS=${1}#" /etc/default/kubelet
 }
 
-function configNetworkPolicy() {
+function configAzureNetworkPolicy() {
+	CNI_VERSION=v0.4.0
```

Comment:
This is the CNI_VERSION that AzureNetworkPolicy uses. Other network policy providers may choose to use other versions. Hence it would be confusing to put it at the top of the script.